rpm/nemo-transferengine-qt5.spec

```diff
@@ -89,8 +88,6 @@ Unit tests for Sailfish Transfer Engine
 %package doc
 Summary:   Documentation for Sailfish Transfer Engine
 License:   BSD
```

Comment:
Interesting. Should we have text for this as well?

Reply:
Commonly we've concentrated on getting the base packages, and the parts installed to device correct. Wouldn't hurt having a separate file for this, of course, but considering that secondary for now.

Reply:
Ok, I'll take it so that this is fine like it is now.